src/server/model_outputs.R

```diff
@@ -81,24 +81,21 @@ makeProgress <- function(n, every = 0.1) {
         counter <<- counter + 1L
         now <- as.numeric(Sys.time())
         if (now - last_time > every) {
-            inc <- (counter - last_counter) / n
+            inc <- ((counter - last_counter) / n)*0.9
```

Comment:
Just cos I'm incrementing it an arbitrary 0.1 amount for the sample drawing stage.